maps/RandomZLevels/wildwest.dm

```diff
@@ -110,66 +264,527 @@
+	icon_state = "elmuerto_2"
+	speed = 3
+	melee_damage_lower = 22
+	melee_damage_upper = 32
+	attacktext = "devours"
+	attack_sound = "sound/weapons/bloodyslice.ogg"
+	items_to_drop = list(/obj/item/weapon/reagent_containers/food/snacks/grown/wheat)
+
+/mob/living/simple_animal/hostile/necro/skeleton/western/heno/death(gibbed)
+	..()
+	qdel(src)
+
+/mob/living/simple_animal/hostile/necro/skeleton/western/heno/enoch
+	name = "Don Enoch"
+	desc = "Cuando el municipio todavía era \"New Zounds\", Don Enoch supervisó la Cámara de Comercio."
+	icon = 'icons/mob/giantmobs.dmi'
+	icon_state = "enoch"
+	icon_living = "enoch"
+	attack_sound = 'sound/weapons/whip.ogg'
+	move_to_delay = 20
+	ranged = 1
+	minimum_distance = 3
+	friendly = "greets"
+	speed = 4
+	maxHealth = 500
+	health = 500
+	harm_intent_damage = 0
+	melee_damage_lower = 5 //Doesn't hit very hard in melee
+	melee_damage_upper = 7
+	ranged_cooldown_cap = 6
+	minimum_shot_distance = 0 //Enoch will always fire his tentacle attack if off cooldown
+	attacktext = "flagellates"
+	size = SIZE_HUGE
+	items_to_drop = list(/obj/item/weapon/reagent_containers/food/snacks/grown/wheat,/obj/item/weapon/reagent_containers/food/snacks/grown/wheat,/obj/item/weapon/reagent_containers/food/snacks/grown/wheat)
+	status_flags = UNPACIFIABLE
+
+/mob/living/simple_animal/hostile/necro/skeleton/western/heno/enoch/OpenFire(atom/ttarget)
+	var/tturf = get_turf(ttarget)
+	if(!istype(tturf, /turf/space) && istype(ttarget))
+		visible_message("<span class='warning'>\The [src] sprouts wheat from below \the [ttarget]!</span>")
+		playsound(loc, 'sound/weapons/whip.ogg', 50, 1, -1)
+		new /obj/effect/goliath_tentacle/original/enoch(tturf)
+		ranged_cooldown = ranged_cooldown_cap
+
+/mob/living/simple_animal/hostile/necro/skeleton/western/heno/enoch/attackby(mob/user, obj/item/I)
+	..()
+	ranged_cooldown_cap--
+	//Cooldown faster when being melee attacked
+
+/mob/living/simple_animal/hostile/necro/skeleton/western/heno/enoch/AfterIdle()
+	say("Now let me get this straight... you come to our town, you trample our crops, you interrupt our private engagement, and now you wanna leave?")
+	say(pick("It saddens me that you don't wish to stay here with us.", "By the authority of the New Zounds Chamber of Commerce, I sentence you to die.", "I find you guilty of trespassing, destruction of property, disturbing the peace, and murder."))
+
+/obj/effect/goliath_tentacle/original/enoch
+	name = "Enoch roots"
+	icon = 'icons/mob/western_mobs.dmi'
+	icon_state = "enoch_tentacle"
+	trip_delay = 1.5 SECONDS
+	spread_dirs = 2
+	//Enoch's tentacles come more often than a goliaths (25%), and trip faster (25%), but spread 1 less direction (100% higher dodge)
+
+/mob/living/simple_animal/hostile/necro/skeleton/western/juergista
+	name = "juergista de la muerte"
+	desc = "Hay quienes no temen a la muerte, pero le dan la bienvenida. Ellos festejan incluso ahora."
```

Comment:
```suggestion
	desc = "Hay quienes no temen a la muerte, sino que le dan la bienvenida. Ellos festejan incluso ahora."
```
The sentence "_Ellos festejan incluso ahora_" rubs me wrong. "_Ellos_" feels redundant but "_Festejan incluso ahora_" feels wrong too. Any other native speakers here that could chip in?